microproxy/layer/application/http1.py

```diff
@@ -15,204 +13,73 @@ def __init__(self, context):
         super(Http1Layer, self).__init__()
         self.context = context
         self.src_conn = Connection(
-            self.context.src_stream, False, self, our_role=h11.SERVER)
+            h11.SERVER,
+            self.context.src_stream,
+            conn_type="src",
```

Comment:
Is there any particular reason to add a conn_type parameter in Connection class.
Or what is the different between the h11.SERVER and conn_type equals "src".